Add docs on get default retention (#740)
timescale/promscale: docs/metric_deletion_and_retention.md

Promscale offers full control over data retentions i.e., you can set a default data retention period as well as overwrite the default on a per-metric basis.

> SQL command to get the default retention policy

Reviewer: ```suggestion
Get the default retention period
```
I think we should keep this as period, since its a duration, than saying a policy.

> ```

Reviewer: ```suggestion
```postgresql
```
Please do the same for the other command. Provides md stylings for language.

SELECT EXTRACT(day FROM _prom_catalog.get_metric_retention_period(''))
```

> SQL command to get the default retention policy for a specific metric

Reviewer: ```suggestion
Get the default retention period, specific to a metric
```
